server/src/api/modules/groups/group.types.ts

```diff
@@ -87,4 +87,8 @@ export type MemberJoinedEvent = Omit<MemberActivity, 'createdAt'>;
 
 export type MemberLeftEvent = Omit<MemberActivity, 'createdAt' | 'role'>;
 
+export type GroupActivityType = MemberActivity & {
```

Comment:
I'd call this `MemberActivityWithPlayer`, it isn't the prettiest, but it's clear and stays consistent with the other types